tiledb/sm/query/reader_base.cc

```diff
@@ -223,6 +223,12 @@ Status ReaderBase::check_validity_buffer_sizes() const {
   return Status::Ok();
 }
 
+bool ReaderBase::skip_missing_timestamps(
```

Comment:
NIT: can we pick a better name? Let's first remove `skip` as we are not skipping but only checking a condition here. Also `missing_timestamps` sounds like an error condition but I think this case is normal, right? Maybe `timestamps_not_enabled` or something of that sort?